### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by sachavskyi**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    files = command.split(" ")
+    for i in range(len(files[2].split("/")[:-1])):
```
Comment (pavlejviki) [inline app/main.py:6]: Use os.makedirs with exist_ok=True to create any missing intermediate directories

File: app/main.py [hunk 2]
```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    files = command.split(" ")
```
Comment (pavlejviki) [inline app/main.py:5]: unpacke the command.split(" ") into relevant variables, making it easier to understand what the command contains.

as well check if the command was correct and consists of three parts

File: app/main.py [hunk 3]
```diff
@@ -1 +1,12 @@
...
+    for i in range(len(files[2].split("/")[:-1])):
+        if not os.path.exists("/".join(files[2].split("/")[:i + 1])):
+            os.mkdir("/".join(files[2].split("/")[:i + 1]))
+    with open(files[2], "w") as f2:
```
Comment (pavlejviki) [inline app/main.py:9]: You can combined the two with open(...) as ...: statements into one

File: app/main.py [hunk 4]
```diff
@@ -1 +1,12 @@
...
+        with open(files[1], "r") as f1:
+            f2.write(f1.read())
+        os.remove(files[1])
```
Comment (pavlejviki) [inline app/main.py:12]: function is a bit hard to follow due to several chained operations and unclear variable names. You can refactor the function to make it more readable and maintainable.

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!
